Created At: 2026-08-25T05:19:35Z
Completed At: 2026-08-25T05:19:36Z

The command completed successfully.
Output:
diff --git a/Refloor/ViewControllers/ToolViewControllers/ToolsSupportedViews/CustomShapeWithLine/CustomShapeLineViewController.swift b/Refloor/ViewControllers/ToolViewControllers/ToolsSupportedViews/CustomShapeWithLine/CustomShapeLineViewController.swift
index 404c5973..1d251a9e 100644
--- a/Refloor/ViewControllers/ToolViewControllers/ToolsSupportedViews/CustomShapeWithLine/CustomShapeLineViewController.swift
+++ b/Refloor/ViewControllers/ToolViewControllers/ToolsSupportedViews/CustomShapeWithLine/CustomShapeLineViewController.swift
@@ -65,6 +65,7 @@ class CustomShapeLineViewController: UIViewController,CustomViewDelegate,LineVie
     var selectedPostion = 0
     var areaSquareFt = 0
     var yAxisLayerSharae:CAShapeLayer!
+    var yAxisLayerLargeDots:CAShapeLayer!
     var xAxisLayerSharae:CAShapeLayer!
     var roomData:RoomDataValue!
     var addData:CustomeShapeViewController!
@@ -95,7 +96,6 @@ class CustomShapeLineViewController: UIViewController,CustomViewDelegate,LineVie
     weak var popupDimmerView: UIView?
     var relocateButton: UIButton!
     
-    
     override func viewDidLoad() {
         super.viewDidLoad()
         //
@@ -123,6 +123,7 @@ class CustomShapeLineViewController: UIViewController,CustomViewDelegate,LineVie
             //        }
             transitionHeightId = transitionHeightDropDownArray[0].transitionHeightId
         }
+        self.setupRelocateButton()
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.3) {
             // Commented out to prevent the scroll view content view from collapsing to 0 width.
             /*
@@ -177,7 +178,6 @@ class CustomShapeLineViewController: UIViewController,CustomViewDelegate,LineVie
             self.no_Squre_View_In_Tool()
             self.setNavigationBarbacklogoResetAndNext(name: self.roomData.name ?? "")
             self.setupNewUI()
-            self.setupRelocateButton()
         }
         
         
@@ -193,11 +193,44 @@ class CustomShapeLineViewController: UIViewController,CustomViewDelegate,LineVie
         }
         
         checkWhetherToAutoLogoutOrNot(isRefreshBtnPressed: false)
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
     }
     
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         IQKeyboardManager.shared.enable = true
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)

